ndv99/canonical-vite-react-template
# Update dependencies

## Done

 - Updated all npm dependencies (except `eslint` core; read below)

`eslint` was not updated at this time, as the migration from v8 to v9 is non-trivial and will require a total rewrite of the configuration file. The existing config and plugins seem to work for now, so I don't believe there's a hard necessity to update to v9 at this time.

## QA

### QA steps

 - `yarn` to install new dependencies
 - Run all `scripts` in `package.json` to ensure they all complete successfully and without error
